src/endpoint-client.ts

```diff
@@ -111,7 +117,7 @@ export class EndpointClient {
 		let axiosConfig: AxiosRequestConfig = {
 			url: this.url(path),
 			method,
-			headers,
+			headers: options?.headerOverrides ? { ...headers, ...options.headerOverrides } : headers,
```

Comment:
Based on how this is written, I'm assuming HTTP headers apply as last one wins. But, can we be sure this is the case across all APIs? Or, is this something being dealt with in axios?

Reply:
Yeah, we can be sure...it's a JavaScript thing and it's fairly common.